src/curiu/ui/cli.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: 2019 The curiu authors
+#
+# SPDX-License-Identifier: LicenseRef-ISC-curiu
```

Comment:
should we always add this header? or we might add it just when we are going to distribute/create releases for it?

Reply:
It might require a little getting used to at first, but it is helpful with the `reuse-tool` to make any license-handling automatic upfront, too. Just like with the code formatting, I'd like to "set and forget"/automate this too :).
(Which reminds me that I have to add `reuse-tool` and its dependencies to conda-forge..)

Reply:
Sure, let's keep in mind to automate and use ``reuse-tool`` then